mutiny-wasm/Cargo.toml

```diff
@@ -25,6 +25,7 @@ lnurl-rs = { version = "0.2", default-features = false }
 wasm-logger = "0.2.0"
 log = "0.4.17"
 gloo-storage = "0.2.2"
+rexie = "0.4"
```

Comment:
Does this have to get pulled in? Probably eventually but not seeing this used yet.

Reply:
oh yeah can remove because I did the test utils thing